sdk/include/reactos/ui/layout.h

```diff
@@ -85,7 +85,7 @@ LayoutShowGrip(LAYOUT_DATA *pData, BOOL bShow)
 
     if (pData->m_hwndGrip == NULL)
     {
-        DWORD style = WS_CHILD | WS_CLIPSIBLINGS | SBS_SIZEGRIP;
+        DWORD style = WS_GROUP | WS_CHILD | WS_CLIPSIBLINGS | SBS_SIZEGRIP;
```

Comment:
```suggestion
        /* CORE-19585: WS_GROUP set to avoid navigation over the grip control */
        DWORD style = WS_GROUP | WS_CHILD | WS_CLIPSIBLINGS | SBS_SIZEGRIP;
```